packages/forgetti/test/__snapshots__/hooks.test.ts.snap

```diff
@@ -35,6 +61,31 @@ function Example(props) {
 }"
 `;
 
+exports[`hooks > should optimize useEffect with 0 dependencies 1`] = `
+"import { useMemo as _useMemo } from \\"react\\";
+import { $$cache as _$$cache } from \\"forgetti/runtime\\";
+import { useEffect } from 'react';
+function Example(props) {
+  let _c = _$$cache(_useMemo, 1),
+    _v = _c[0] ||= [];
+  useEffect(() => props.value(), [_v]);
+}"
+`;
```

Comment:
IMHO, useEffect with 0 dependency doesn't need to be transformed at all.

Reply:
0 dependency transformation means auto-dependency transformation. There's no way to automatically track dependencies for `effect` hooks at the moment, and this change practically aligns to `callback` and `memo` hooks

Reply:
Same reason as the previous comment.

Reply:
I see, thanks!